### PR TITLE
finish bandit integration

### PR DESCRIPTION
**Pull request by l1l1l1l1l1l1l**
bandit integration finished
except I cannot find ways to put my tests file(vulns_tests.py & trueResult.html) into a separate folder while keeping it works

### CODE REVIEW

File: checker/cli.py [hunk 1]
```diff
@@ -39,6 +171,10 @@ def sub_command_repo_commit(args):
     GitRepo(args.basepath).print_commit_info()
 
 
+@command('vulns')
+def sub_command_vulns(args):
+    main(args)
```
Comment (xzhang2437) [inline checker/cli.py]: Please try another function name instead of _main_.

File: test/trueResult.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
+
```
Comment (xzhang2437) [inline test/trueResult.html]: Could you please move this file to **_test/resources/_** ?

Comment (l1l1l1l1l1l1l) [reply test/trueResult.html]: moved to test/resources/vulns

File: checker/cli.py (hunk 1 above)
Comment (l1l1l1l1l1l1l) [reply checker/cli.py]: checked to vulnsCheck

File: checker/cli.py [hunk 3]
```diff
@@ -20,6 +26,132 @@
...
+                                                hasattr(x.plugin,
+                                                        '_accepts_baseline'),
+                                                extension_mgr.formatters)]
+VULNS = SUBPARSERS.add_parser('vulns',
+        help='find vulnerabilities in repo',
+        formatter_class=argparse.RawDescriptionHelpFormatter
+)
+VULNS.add_argument(
+    'targets', metavar='targets', type=str, nargs='*',
+    help='source file(s) or directory(s) to be tested'
```
[the remark this answers is not in the document]
Comment (l1l1l1l1l1l1l) [reply checker/cli.py]: seems it's already required, see [ref](https://stackoverflow.com/questions/45078474/required-is-an-invalid-argument-for-positionals-in-python-command)

**Comment by l1l1l1l1l1l1l** [comment]
finished

**Pull request merged**